lib/utils/schema/inspec.exec.min.json

```diff
@@ -0,0 +1 @@
+{"type":"object","additionalProperties":false,"properties":{"statistics":{"type":"object","additionalProperties":false,"properties":{"duration":{"type":"number"}}},"version":{"type":"string"},"controls":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"id":{"type":"string"},"profile_id":{"type":["string","null"]},"status":{"type":"string"},"code_desc":{"type":"string"},"skip_message":{"type":"string","optional":true},"resource":{"type":"string","optional":true}}}}}}
```

Comment:
Same comment as inspec.exec.full.json - would love to see this structured so humans can grok it.